server/controllers/exporters/xlsx.js

```diff
@@ -0,0 +1,107 @@
+const xlsx = require("node-xlsx");
+
+const parseGeneNameArray = function (genes) {
```

Comment:
Just a naming note; the `parse` functions here aren't really parsing—they're actually the opposite of parsing. These are more like "converting" or "building" parts of the sheet.

It's all just naming, but admittedly I had to double-take a bit when I saw that there was "parsing" going on until I realized that it actually pertained to converting/building/creating parts of an xlsx.

Reply:
Unless I'm misunderstanding/misinterpreting the intent of these functions?

Reply:
Nono that's fair! Was misconstruing the idea of parsing. Will either use "convert" or "build" for the names of these functions.

Reply:
Updated!